lib/disc.rb

```diff
@@ -101,7 +109,8 @@ def disque=(disque)
         @disque = disque
       end
 
-      def disc(options = {})
+      def disc(queue: nil, **options)
+        @queue = queue
```

Comment:
Yeah, that's me copying the logic from the disque class method without thinking. Using `@queue || Disc.default_queue` should fix that case.